### PR TITLE
[doc] add documentation about reducing traffic

### PR DESCRIPTION
**Pull request by jonasschnelli**
We have seen node operators shutting down nodes because of traffic exceeding. Reducing traffic is possible with different measures and therefore very likely a "communication problem". This documentation might help those who like to or need to reduce traffic.

Thanks @fanquake for the overhaul!

### CODE REVIEW

**Comment by jonasschnelli** [comment]
Release notes for maxuploadtarget is handled over #6958.

**Comment by MarcoFalke** [comment]
Looks good. Concept ACK

File: doc/reducetraffic.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+REDUCE TRAFFIC
```
Comment (MarcoFalke) [inline doc/reducetraffic.md]: Nit: No CAPSLOCK

File: doc/reducetraffic.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+during the initial blocks download phase (syncing up a new node).
+This option can be specified in MiB per day and is turned off by default.
+This is *not* a hard limit; only a threshold to minimize the outbound
+traffic. When the limit is about to be reached, the uploaded data is cut by no
+longer serving historic blocks (blocks older than one week).
+Keep in mind that new nodes require other nodes that are willing to serve
+historic blocks. **The recommended minimum is 144 blocks per day (max. 144MB
+per day)**
+
+2. Disable "listening" (`-listen=0`)
```
Comment (MarcoFalke) [inline doc/reducetraffic.md]: Nit: .md will convert this to `1.`. Use subsections?

File: doc/reducetraffic.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+A major component of the traffic is caused by serving historic blocks to other nodes
+during the initial blocks download phase (syncing up a new node).
+This option can be specified in MiB per day and is turned off by default.
+This is *not* a hard limit; only a threshold to minimize the outbound
+traffic. When the limit is about to be reached, the uploaded data is cut by no
+longer serving historic blocks (blocks older than one week).
+Keep in mind that new nodes require other nodes that are willing to serve
+historic blocks. **The recommended minimum is 144 blocks per day (max. 144MB
+per day)**
```
Comment (MarcoFalke) [inline doc/reducetraffic.md]: Nit: Don't overuse bold. Make only a single word bold?

**Comment by jonasschnelli** [comment]
Fixed @MarcoFalke's nits. The bold test part is okay IMO.

File: doc/reducetraffic.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Disabling listening will result in fewer nodes connected (remember the maximum of 8
+outbound peers). Fewer nodes will result in less traffic usage as you are relaying
+blocks and transactions to fewer nodes.
+
+## 3. Reduce maximum connections (`-maxconnections=<num>`)
+
+Reducing the maximum connected nodes to a miniumum could be desirable if traffic
+limits are tiny. Keep in mind that bitcoins trustless model works best if you are
+connected to a handfull of nodes.
```
Comment (carnesen) [inline doc/reducetraffic.md]: *handful

Comment (jonasschnelli) [inline doc/reducetraffic.md]: Thanks! Force-Push fixed.

**Comment by jtimon** [comment]
ACK

**Comment by jmcorgan** [comment]
utACK

**Comment by gavinandresen** [comment]
ACK.

(Nit: I don't like the use of the passive voice; for example "Reducing the maximum connected nodes to a miniumum could be desirable" would be clearer as  "Set -maxconnections=1 for minimal bandwidth usage" ... but I think this is good enough to pull as-is).

**Pull request merged**